<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2</class>
 <widget class="QDialog" name="Dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>365</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>64</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="donate">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>202</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>47</width>
     <height>147</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>510</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/assets/adblink.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="linkLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>275</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://www.jocala.com&quot;&gt;&lt;span style=&quot; font-size:12pt; text-decoration: underline; color:#0000ff;&quot;&gt;http://www.jocala.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="thankLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>275</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Thanks for your donations!</string>
   </property>
  </widget>
  <widget class="QLabel" name="alabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>275</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>adbLinkX 9999</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="betaLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>318</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BETA</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog2</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
